docs/Installation.md

```diff
@@ -0,0 +1,101 @@
+# Installation Guide for ClusterLink
+The ClusterLink gateway contains three main components: the control-plane, data-plane, and gwctl (more details can be found [here](../README.md#what-is-clusterlink)).
+
+ClusterLink can be deployed in a local environment (e.g., Kind) or any remote K8s cluster environment (e.g., google GKE, amazon EKS, IBM IKS, etc.).
+
+To deploy the ClusterLink gateway in a K8s cluster, follow these steps:
+
+1. Create and deploy certificates and the ClusterLink deployment YAML file.
+2. Expose the ClusterLink deployment to a public IP.
+3. Optionally, create a central gwctl component to control one or more gateways.
+
+Before you begin, please build the project according to the [instructions](../README.md#building-clustelink).
+### 1. ClusterLink deployment and certificates
```

Comment:
nit: 
- be inconsistent with other headings (e.g.,  `Installation Guide for ClusterLink` above)
- skipping two heading levels (from `#` to `###`)?